catalog/spatial/spatial-commands/src/main/java/org/codice/ddf/commands/spatial/geonames/GeoNamesUpdateCommand.java

```diff
@@ -14,12 +14,11 @@
 package org.codice.ddf.commands.spatial.geonames;
```

Comment:
Rename happy... I think this one is already done and will be picked up on merge

Reply:
Already renamed